lib/commands/envs.ts

```diff
@@ -181,24 +184,59 @@ export default class EnvsCmd extends Command {
 			return i;
 		});
 
-		if (options.device) {
-			fields.push(options.json ? 'deviceUUID' : 'deviceUUID => DEVICE');
-		}
-		if (!options.config) {
-			fields.push(options.json ? 'serviceName' : 'serviceName => SERVICE');
-		}
+		if (isV14()) {
+			const results = [...varArray] as any;
 
-		if (options.json) {
-			const { pickAndRename } = await import('../utils/helpers');
-			const mapped = varArray.map((o) => pickAndRename(o, fields));
-			this.log(JSON.stringify(mapped, null, 4));
+			// Rename fields
```

Comment:
Dropping support for the old cli-visuals field renaming syntax required a bit more code here, but it's only used in a couple of places in the codebase.

Reply:
I was going to suggest just dropping the renaming, like I think you had suggested before, as it is more friendly to code autogeneration and this change only applies to v14 (OK to be a breaking change).
But then I tested it and realised that `cli-ux` does not play well with camelCase, e.g. `deviceUUID` became `Deviceuuid` in the table header:
```
$ ./bin/balena-dev envs -d a459e1677ba853e10af2f655faaca503
 Id      Name          Value            Fleet           Deviceuuid           Service
...
```
I think this means we need to add some code somewhere for `cli-ux` (or the CLI codebase) to automatically convert camelCase to spaces, e.g. `deviceUUID` becomes `Device UUID`. And if we were to do that, then we might as well consider (optional) copying the existing `pickAndRename` logic (`helpers.ts`) that understands renaming specifications like 'deviceUUID => DEVICE'.

What do you think?

Reply:
Yeah ok, I think dropping the renaming is best if it works.  Adding that camelCase processing will be quite straightforward.  I think perhaps it would be best not to carry over the old conventions (at least initially) to keep things lean.  Since we don't do renames much, I think renaming manually in the logic is a bit more explicit.  It might also push us to modify the model as needed rather than patching in the view.  We can always add support for the rename conventions later (though the pickAndRename logic won't be a drop-in, I did take a look at it and it will need some modification).